clang/test/SemaCXX/attr-annotate.cpp

```diff
@@ -1,4 +1,64 @@
-// RUN: %clang_cc1 -std=gnu++20 -fsyntax-only -verify %s
+// RUN: split-file %s %t
+// RUN: %clang_cc1 -std=gnu++20 -fsyntax-only -ast-dump -verify %t/good_annotate.cpp | FileCheck %s
```

Comment:
This looks like it should just be a separate file instead, `attr-annotate-ast.cpp` instead of this split file stuff is much preferred.  Also, the `bad` line has no reason to do `verify`, you aren't actually checking diagnostic lines, just doing an AST dump.